### DISCUSSION TITLE
The end of PyCaret

### DISCUSSION THREAD

**Discussion by celestinoxp** [pycaret/pycaret#4087] [General]
As everyone has seen, pycaret has come to an end... the team abandoned the project (personal and job changes). So it's time to move to a project that has constant maintenance and code updated with the latest packages...

I'm currently migrating my projects to Autogluon, from what I understand it's the most similar to Pycaret I've found (low code) and it seems to work very well.

What do you think about this? What is your opinion?

**Comment by pytoned** [comment]
So sad... loved pycaret 😓

**Comment by falconair** [comment]
Is there an official announcement about this or did the maintainers just abandon the project? Any discussion about anyone taking it over?

**Reply by Artem-Potapov** [reply]
Doesn't seem so. And people would likely take over already, because so many loved it (including me). So yeah, I guess it's time to say our goodbyes and move on.. 
Or maybe someone could fork it

**Reply by jakob1379** [reply]
fork would be nice, but an actual handover with new maintainers would likely be the best option, though not likely.
